### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_AllJobs_ScreenReader:NVDA announcing the data from "Overview" tab in collapse state.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5748]
**Bug Description:**
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.2; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details:
Tool Name:NVDA
Version :2019.1
MAS Reference:MAS1.3.2
**Repro Steps:**
1.Install the app Azure Data Studio.
2.Naviagte to the SQL agent tab->Jobs->Click on any the jobs in the table.
3.Navigate till overview tab.
4.Start NVDA
5.observed the announcement of screen reader (NVDA)
**Actual:**
NVDA is announcing inside data from the "overview" tab when it is in collapse state.
![image](https://user-images.githubusercontent.com/39204909/58860125-0e6bb900-86c9-11e9-987a-dcf1a84bfc60.png)

**Expected:**
NVDA should announce the data in logical order so that user will find smooth navigation.
**User Impact:**
User with vision and motor disabilities will be confuse if the screen reader will not read the data in logical order.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:1.3.2**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}
**Attachment for Reference:**
[5748_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3237004/5748_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes
